front/app/component-library/utils/testUtils/rtl.tsx

```diff
@@ -2,17 +2,20 @@
 import React, { FC } from 'react';
 
 import { render, RenderOptions } from '@testing-library/react';
+import { HelmetProvider } from 'react-helmet-async';
 import { ThemeProvider } from 'styled-components';
```

Comment:
`react-helmet` is now considered deprecated, and it's recommended to use `react-helmet-async` instead :)